docusaurus.config.js

```diff
@@ -154,7 +154,7 @@ const config = {
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} Conda Community. <br /> Built with Docusaurus. This site is powered by <a href="https://www.netlify.com/" target="_blank" title="Powered by Netlify">Netlify</a>.`,
+        copyright: `Copyright © ${new Date().getFullYear()} Conda Community. <br /> Built with Docusaurus. This site is powered by <a href="https://www.netlify.com/" style="color: #d6eed1" target="_blank" title="Powered by Netlify">Netlify</a>.`,
```

Comment:
Should this be slightly green or is it sufficient with just white? The rest of hyperlinks the footer and just white.

Reply:
Ok, then I guess the footer links need to be updated to match. Right now there's an inconsistency:

Normal:

<img width="319" alt="image" src="https://github.com/user-attachments/assets/23ca28a1-e4bc-4e4f-93b3-da6dd7c82d37" />

Footer hover:

<img width="277" alt="image" src="https://github.com/user-attachments/assets/11df5c7f-33be-4d6d-8543-e41d59c90a23" />


Copyright hover:

<img width="268" alt="image" src="https://github.com/user-attachments/assets/3226141c-342e-470d-bd0f-cd2960f8d4d4" />

Reply:
Good catch, I missed this as I had to add the colour change in place.